docs/current/sdk/nodejs/835948-install.md

```diff
@@ -0,0 +1,5 @@
+---
+slug: /sdk/nodejs/835948/install
+---
+
+# Installation
```

Comment:
Are we recommending yarn or npm? Do we already have the package name?

Reply:
I would suggest both npm and yarn

Reply:
> Are we recommending yarn or npm?

Either both or if only one that should be `npm`

> Do we already have the package name?

`@dagger.io/dagger` to be consistent with `dagger.io/dagger` (Go) and `dagger-io` (Python)